src/server/troubleshoot/index.md

```diff
@@ -41,4 +41,13 @@ Logs should contain information about sending emails with task `mergin.celery.se
 
 ```shell
 [2024-12-09 10:37:16,265: INFO/ForkPoolWorker-2] Task mergin.celery.send_email_async[3e50df69-90c1-49be-b31c-78f1fb417500] succeeded in 0.11469305199989321s: None
-```
\ No newline at end of file
+```
+
+## Celery settings
+
+The following section provides some background to Celery setting via available environment variables. All these variables start with `CELERY_`
+
+* `CELERY_RESULT_BACKEND` - Where Celery sends the tasks results (to Redis backend by default on Mergin Maps)
```

Comment:
More than result backend (we do not really use tasks results) we should mention BROKER_URL and maybe some words about task queue .